docs/specs/perception_apollo_5.0.md

```diff
@@ -55,6 +61,30 @@ The CIPO includes detection of key objects on the road for longitudinal control.
 ![](images/CIPO_2.png)
 
 
+### Vanishing Point Detection
+
+In Apollo 5.0, an additional branch of network is attached to the end of the lane encoder to detect the vanishing point. This branch is composed of convolutional layers and fully connected layers, where convolutional layers translate lane features for the vanishing point task and fully connected layers make a global summary of the whole image to output the vanishing point location. Instead of giving an output in `x`, `y` coordinate directly, the output of vanishing point is in the form of `dx`, `dy` which indicate its distances to the image center in `x`, `y` coordinates. The new branch of network is trained separately by using pre-trained lane features directly, where the model weights with respect to the lane line network is fixed. The Flow Diagram is included below, note that the red color denotes the flow of the vanishing point detection algorithm.
+
+![](images/Vpt.png)
```

Comment:
I could not see the image inserted from the .md view, is there anything wrong

Reply:
I mean in the rich diff mode, other images can be shown, but the newly inserted can not.

Reply:
@yg13 the images are still part of the PR. You will be able to see them after the PR was made. Please refer to [this link](https://github.com/natashadsouza/apollo/blob/master/docs/specs/perception_apollo_5.0.md) to see a preview. Thanks!